homeassistant/components/tessie/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "tessie",
+  "name": "Tessie",
+  "codeowners": ["@Bre77"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/tessie",
+  "iot_class": "cloud_polling",
+  "loggers": ["tessie"],
+  "requirements": ["tessie-api==0.0.2"]
```

Comment:
Maybe peg to `0.0.8` so your [Keep Mode](https://github.com/andrewgierens/tessie_python_api/pull/2) changes are included?

Reply:
Cool, didnt see you had pushed a new release. I will do so.

Reply:
Blocked by https://github.com/andrewgierens/tessie_python_api/pull/6

Reply:
Your change was merged and deployed here - I swear I fixed this but must have just been locally!
https://github.com/andrewgierens/tessie_python_api/releases/tag/0.0.9

Reply:
Bumped to 0.0.9.
Would appreciate the review approval once you can, as the core team doesnt seem to review PRs until they have at least 1 review approval.

Reply:
Done :)